docker-compose.yml

```diff
@@ -0,0 +1,24 @@
+version: '3'
+services:
+  app:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    container_name: Grasa-Django-App
+    network_mode: host
+    ports:
+      - 80:80
+      - 443:443
+  db:
+    image: mysql:latest
+    command: mysql --default-authentication-plugin=mysql_native_password
+    volumes:
+      - "./mysql:/var/lib/mysql"
```

Comment:
Is there a `$TMPDIR` we can use to avoid storing files in the git repo, e.g. `/tmp/grasa_event_locator:/var/lib/mysql`?

Also, it is not obvious, but for systems using SELinux like CentOS 7, you need to [append an SELinux flag](https://www.projectatomic.io/blog/2015/06/using-volumes-with-docker-can-cause-problems-with-selinux/) to the volume string with `:z`.

Reply:
using a directory in /tmp isn't a good idea because files in that directory are deleted on reboot.  In order to keep our application data persistent, we should use a non /tmp directory, such as /grasa/db/

Reply:
@nlMeminger I guess it depends whether you want your database to be ephemeral or not :slightly_smiling_face: We can go with this for now, but don't forget the `:z` SELinux volume flag.

Reply:
@jwflory works for me